lib/util/git/index.ts

```diff
@@ -551,6 +551,30 @@ export async function isBranchModified(branchName: string): Promise<boolean> {
   return true;
 }
 
+export async function isBranchConflicted(
```

Comment:
If this command is called in both the branch worker and PR worker, we should find a way to cache the result, either in the git layer or by adding the result to the branch config that's passed to the PR worker.